### PR TITLE
964 Roll out logging in repository

### PR DESCRIPTION
**Pull request by jasminziegler**
# Description
Logging concept has been approved on 
- [x] CNNDepthMap/CNNDepthMap-height/q3-depthmap-plaincnn-height
and is now ready for roll out in the repository.
For a start, all files in cgm_ml/src/models will be updated.

related to #274 #148 
Component Requirement https://dev.azure.com/cgmorg/ChildGrowthMonitor/_sprints/backlog/ChildGrowthMonitor%20Team/ChildGrowthMonitor/2021.Q1/Q1.Sprint%203?workitem=964

# Model Folders to be updated
CNNDepthMap
- [x] CNNDepthMap-height/q2milestone-cnndepthmap-height
- [x] CNNDepthMap-height/q2s1-cnndepthmap.height
- [x] CNNDepthMap-height/q3-cnndepthmap-resnet-height
- [x] CNNDepthMap-height/q3-depthmapmultiartifact-plaincnn-height
- [x] CNNDepthMap-height/q3-depthmapmultiartifactlatefusion-plaincnn-height
- [x] CNNDepthMap-height/s4-cnndepthmap-height
- [x] CNNDepthMap-height/s5-cnndepthmap-height
- [x] CNNDepthMap-multitask/q4-depthmap-plaincnn-multitask
- [x] CNNDepthMap-weight/s5-cnndepthmap-weight

CNNRGB
- [x] CNNRGB-standing_laying/q4-rgb-standing-laying

GAPNet
- [x] GAPNet-height/s1-gapnet-height
- [x] GAPNet-height/s2-gapnet-height

PersonLab
- [x] PersonLabV1

pointnet
- [x] pointnet-height/pointnet-20190708
- [x] pointnet-height/pointnet-20190806

Unsupervised
- [x] Autoencoder/q4-anomalydetection-ae

# How Has This Been Tested?

Models above are only ticked if logs work on compute cluster during training.
- run train.py locally
- run training scripts with nbr of epochs = 2
- check logging output in azureml_logs/70_driver_log.txt
- in some cases, training scripts could not be run bec. model not updated after moving tenants, model not curated, etc
- since the logging statements worked in most of the files, it can be assumed that it also works in the others 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] My Code is review by 1 People

### CODE REVIEW

File: src/models/CNNDepthMap/CNNDepthMap-height/q3-depthmapmultiartifact-plaincnn-height/src/preprocessing_multi.py [hunk 1]
```diff
@@ -113,8 +116,7 @@ def create_multiartifact_sample(artifacts: List[str]) -> Tuple[tf.Tensor, tf.Ten
         targets_list.append(targets)
     targets = targets_list[0]
     if not np.all(targets_list == targets):
-        print("Warning: Not all targets are the same!!\n"
-              f"target_list: {str(targets_list)} artifacts: {str(artifacts)}")
+        logging.info('Warning: Not all targets are the same!! \n' + 'target_list: %s', targets_list + 'artifacts: %s', artifacts)
```
Comment (jasminziegler) [inline src/models/CNNDepthMap/CNNDepthMap-height/q3-depthmapmultiartifact-plaincnn-height/src/preprocessing_multi.py:119]: wrong syntax. changed this

File: src/models/CNNDepthMap/CNNDepthMap-height/q3-depthmapmultiartifactlatefusion-plaincnn-height/src/preprocessing_multiartifact.py [hunk 2]
```diff
@@ -37,8 +41,7 @@ def create_multiartifact_sample(artifacts: List[str]) -> Tuple[tf.Tensor, tf.Ten
         targets_list.append(targets)
     targets = targets_list[0]
     if not np.all(targets_list == targets):
-        print("Warning: Not all targets are the same!!\n"
-              f"target_list: {str(targets_list)} artifacts: {str(artifacts)}")
+        logging.info('Warning: Not all targets are the same!! \n' + 'target_list: %s', targets_list + 'artifacts: %s', artifacts)
```
Comment (jasminziegler) [inline src/models/CNNDepthMap/CNNDepthMap-height/q3-depthmapmultiartifactlatefusion-plaincnn-height/src/preprocessing_multiartifact.py:44]: changed syntax

File: src/models/Unsupervised/Autoencoder/q4-anomalydetection-ae/src/config.py [hunk 3]
```diff
@@ -78,7 +78,7 @@
     LATENT_DIM=2,
 
     KL_LOSS_FACTOR=0.5,
-    EPOCHS=40,
+    EPOCHS=2,
```
Comment (jasminziegler) [inline src/models/Unsupervised/Autoencoder/q4-anomalydetection-ae/src/config.py:81]: removed this temporary change, I used it for testing

**Review by jasminziegler — COMMENTED** [review]
Going to apply some minor changes in upcoming push that I noticed during self review.

File: src/models/CNNDepthMap/CNNDepthMap-height/q3-cnndepthmap-resnet-height/code/utils.py [hunk 4]
```diff
@@ -65,9 +69,9 @@ def make_grid(image_dir):
         y_position = current_file_number // grid_size
         plt_image = plt.imread(image)
         axes[x_position, y_position].imshow(plt_image)
-    # print((current_file_number + 1), '/', images_count, ': ', image_filename)
+    #logging.info('%d / %d: %s', (current_file_number+1), images_count, images_filename)
 
-        current_file_number += 1
+    current_file_number += 1
```
Comment (jasminziegler) [inline src/models/CNNDepthMap/CNNDepthMap-height/q3-cnndepthmap-resnet-height/code/utils.py:74]: change this in next push

**Comment by markus-hinsche** [comment]
Looks good at first sight, please assign me to a review when the PR is ready

**Comment by markus-hinsche** [comment]
Please solve the merge conflict

**Review by markus-hinsche — APPROVED** [review]
LGTM

**Pull request merged**